### PR TITLE
fix(curriculum): improve the wording of a challenge

### LINKED ISSUE #45629: in here should exist <code></code> element
## Affected page

[https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-accessibility-by-building-a-quiz/step-17](url)
## Your code

```
`<code>sans-serif</code>
`
```

## Expected behavior

in here should exist <code></code> element:
`<code>sans-serif</code>
`
## Screenshots
image

## System

win11  Firefox
- Device: [e.g. iPhone6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.4]
- Browser: [e.g. chrome, safari]
- Version: [e.g. 22]
 
## Additional context

I want to modify it myself，but I don't know how to deal with it. I don't know how to find him

### PR DESCRIPTION
**Pull request by yuanhepeng**
make the wording clearer

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x ] My pull request targets the `main` branch of freeCodeCamp.
- [ x] I have tested these changes either locally on my machine, or GitPod.

Closes #45629

### CODE REVIEW

**Comment by yuanhepeng** [comment]
em...I want to change to "Request changes"...
image

**Review by huyenltnguyen — DISMISSED** [review]
LGTM

**Comment by raisedadead** [comment]
You are unable to request changes as the author. The idea way is you should work on your local copy (your machine, GitPod) and push them up before creating a PR. If you want to add more changes to a give PR, you can add more changes, create more commits and push them to the same branch.

Please check out our [contributing guidelines](https://contribute.freecodecamp.org) and GitHub's docs if you need more help or reach out to the [contributors chat room](https://chat.freecodecamp.org/channel/contributors).

**Review by raisedadead — DISMISSED** [review]

**Comment by yuanhepeng** [comment]
I just added a link, points to an article，I want to explain the concept more clearly

**Comment by yuanhepeng** [comment]
I want to open this article in a new tab, I tried target="_blank" and rel="external", but it doesn't seem to have achieved the effect.

**Comment by moT01** [comment]
I kind of lean towards just improving the wording like was discussed in the issue. I'm not completely against the link if others are in favor of it, but we usually try to avoid links in the descriptions/instructions. On rare occasions, we do use them - I don't think it's needed here, though.

**Comment by huyenltnguyen** [comment]
@moT01 Agreed. We are sending the user to an article in this case, and the user would need to consume the content and scroll a bit further down to find the font list, which might distract them from the challenge. We could add a small section to the challenge description for a list of fonts the user could from, but I'm not really sure if that is really needed.

File: curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/614206033d366c090ca7dd42.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ dashedName: step-17
 
 Typeface plays an important role in the accessibility of a page. Some fonts are easier to read than others, and this is especially true on low-resolution screens.
 
-Change the font for both the `h1` and `h2` elements to `Verdana`, and use another sans-serif _web safe_ font as a fallback.
+Change the font for both the `h1` and `h2` elements to `Verdana`, and use another web-safe font in the <a href="https://www.freecodecamp.org/news/html-font-css-font-family-example-serif-and-sans-serif-characters/" rel='external'>sans-serif family</a> as a fallback.
```
Comment (nhcarrigan) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/614206033d366c090ca7dd42.md:12]:
```suggestion
Change the font for both the `h1` and `h2` elements to `Verdana`, and use another web-safe font in the sans-serif family as a fallback.
```
Per above 🙂

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Comment by huyenltnguyen** [comment]
@yuanhepeng Thanks for addressing our feedback! 

I approved the other PR, but just wanted to put a note here 🙂 You shouldn't need to create a new PR to update the change. If you clicked the "Commit suggestion" button in @nhcarrigan's https://github.com/freeCodeCamp/freeCodeCamp/pull/45668#discussion_r848512888, a new commit would be added to the branch with the suggested change. 

Anyway, thank you for your patience and your contribution to freeCodeCamp!

**Pull request closed without merging**